Equipment/EquipmentDb/Gloves/leather.xml

```diff
@@ -238,6 +238,18 @@
 		</source>
 	</item>
 
+	<item id="21178" phase="5">
+		<info name="Gloves of Earthen Power" type="LEATHER" slot="GLOVES" unique="no" req_lvl="0" item_lvl="62" quality="RARE" boe="no" icon="Inv_gauntlets_18.png"/>
```

Comment:
The required level for quest rewards is the minimum level that the quest can be completed at, which in this case appears to be 54.

Reply:
Thanks, i'll doublecheck that on future items.